tests/specs/check/check_node_builtin_modules_ts/__test__.jsonc

```diff
@@ -0,0 +1,9 @@
+{
+  "args": "check --quiet node_builtin_modules/mod.ts",
+  "output": "node_builtin_modules/mod.ts.out",
+  "envs": {
+    "NO_COLOR": "1",
+    "NPM_CONFIG_REGISTRY": "http://localhost:4260/"
+  },
```

Comment:
```suggestion
```

You should be able to remove these. They get automatically set for the spec tests.

Reply:
I opened https://github.com/denoland/deno/pull/25982 to fix these on main